src/kw_env.sh

```diff
@@ -66,9 +67,14 @@ function use_target_env()
     fi
 
     # Create symbolic link
-    ln --symbolic --force "${local_kw_configs}/${target_env}/${config}.config" "${local_kw_configs}/${config}.config"
+    ln --symbolic --force "${local_kw_configs}/${ENV_DIR}/${target_env}/${config}.config" "${local_kw_configs}/${config}.config"
   done
 
+  # Restore kernel .config from env
+  if [[ -e "${local_kw_configs}/${ENV_DIR}/${target_env}/.config" ]]; then
+    cp "${local_kw_configs}/${ENV_DIR}/${target_env}/.config" .config
+  fi
```

Comment:
Actually, you don't want to copy `.config` inside `.kw` folder; you want it on `${KW_CACHE_DIR}/${target_env}`. Kw env will create a folder in the cache path for the new env, and all products of the kernel compilation will be isolated in the cache path. Also, since `.config` file is specific to each env, we must keep it inside the env.